### ISSUE TITLE
Settings Icons

### ISSUE THREAD

**Issue by EricBAndrews** [mlemgroup/mlem#1629]
V1 feature; settings entries should have associated icons to assist in dyslexic users (or others who have trouble picking words out of long lists of text)

**Comment by Sjmarf** [comment]
In v1 I believe this was on by default? Personally I'm leaning toward keeping it off by default. 

Having a lot of icons does make settings easier to find if there's a lot of settings in one page. But, IMO it would be better to fix that by just reducing the number of settings per page, and adding diagrams etc to make the settings more obvious. 

Apple themselves don't have icons in Settings. Adding icons reduces the amount of space we have horizontally on each row, especially on smaller devices. This makes it difficult to fit certain information in (e.g. the navigation links with a value such as the "Notification Badge" setting link).

**Comment by EricBAndrews** [comment]
I'm on board with default off--it's definitely cleaner and "more Apple."

> Apple themselves don't have icons in Settings.

True, but we've gotten some really positive feedback from dyslexic users on having this feature, so I'm content to break from Apple design on this front.
